src/litserve/loops.py

```diff
@@ -505,7 +610,7 @@ def __call__(
         run_single_loop(lit_api, lit_spec, request_queue, response_queues, callback_runner)
 
 
-class BatchedLoop(_BaseLoop):
+class BatchedLoop(DefaultLoop):
```

Comment:
since it is used as base, maybe rename `DefaultLoop` to `BasetLoop`?

Reply:
`DefaultLoop` has a specific `pre_setup` method which BaseLoop shouldn't have